src/options.rs

```diff
@@ -742,35 +742,35 @@ fn get_default_widget_and_count(
 fn get_use_battery(matches: &ArgMatches, config: &Config) -> bool {
     #[cfg(feature = "battery")]
     {
-        if let Ok(battery_manager) = Manager::new() {
-            if let Ok(batteries) = battery_manager.batteries() {
-                if batteries.count() == 0 {
-                    return false;
-                }
-            }
-        }
-
         if matches.get_flag("battery") {
             return true;
         } else if let Some(flags) = &config.flags {
             if let Some(battery) = flags.battery {
                 return battery;
             }
         }
+
+        if let Ok(battery_manager) = Manager::new() {
+            if let Ok(batteries) = battery_manager.batteries() {
+                if batteries.count() == 0 {
+                    return false;
+                }
+            }
+        }
```

Comment:
> why this change?

The change was made for the 'battery config/cli precedence' to show the widget regardless of detected batteries. I was using it for the gpu power and the tab layout bounds when without a battery. 

This was a future change suggested in the PR that ignored the battery widget without detected batteries. 

> #975 
"If needed, it can be adjusted so that if the config file contains the battery field, it will still show the widget even without batteries on the system."

The suggested change:
> more like:
`if use_battery {
  if let Ok(battery_manager) = Manager::new() {
    // ...
  }
} else {
  false
}`

would again ignore the no battery present but still want the widget case as the commented out code above is
`if batteries.count() == 0 {
                    return false;
}`